src/main.cpp

```diff
@@ -1299,8 +1299,7 @@ bool VerifyAmounts(const CCoinsViewCache& cache, const CTransaction& tx, std::ve
                 if (!MoneyRange(val.GetAmount()))
                     return false;
 
-                if (val.GetAmount() == 0)
-                    continue;
+                assert(val.GetAmount() != 0);
```

Comment:
If you try to make a commitment of value 0 with no blinder, I think it hits a libsecp assert.